README.md

```diff
@@ -1,25 +1,115 @@
 # glazier
 
-`glazier` is an attempt to provide a common interface to the various
-elements of different platform application frameworks. It is designed to be used
-by [druid], a UI toolkit.
+Glazier is a portable infrastructure layer intended for high quality GUI
+toolkits in Rust. It is based on druid-shell but with some changes, primarily
+around making it agnostic to the choice of drawing.
+
+Primary platforms are Windows, macOS, and Linux. Web is a secondary platform.
+Other ports may happen if they are contributed.
+
+This library is currently work in progress and should be considered experimental.
+Contributions are welcome, see [CONTRIBUTING](./CONTRIBUTING.md) for more details.
+
+## Scope
+
+The following tasks are in scope. Mostly they are implemented, but as always
+there is more refinement to be done.
+
+* Window creation, including subwindows.
+
+* System menus. These are especially important on macOS.
+
+* Keyboard events.
+
+* Input Method Editor. On macOS, correct handling of dead keys is through the
+IME, and an application built on Glazier is expected to handle IME.
+
+* Mouse and pointer events. Mouse events are currently supported, pointers
+are a hoped-for feature.
```

Comment:
Is mouse events the raw input, and pointer the motion of the cursor? Winit uses the terminology mouse and cursor for this (e.g. [`winit::event::WindowEvent::CursorMoved`](https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html#variant.CursorMoved))

It seems to me that correctly handling pointer events seems much more important than handling mouse events, but that disagrees with the second sentence. I haven't dug into the code to see where this goes wrong.

Reply:
By pointer I mean touch and pen. https://github.com/linebender/druid/pull/1965 is a draft implementation in the druid repo, never landed (and I haven't had a chance to look at it in detail). I did some research years ago, wrote up some [notes](https://docs.google.com/document/d/1QxgqL52aXNJASrq4mUVivDTJOwy-QzobXz7X7QVzfnk/edit).

Reply:
Ah, reasonable. I wasn't familiar with pointers having that meaning.

Since both terms are quite jargony, it's probably better to mention both and clarify (depending on the audience you want to reach)